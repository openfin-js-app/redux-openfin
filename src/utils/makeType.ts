export const PREFIX:string = 'REDUX_OPENFIN';
export const DELIMITER:string = '::';

export default (type:string) => `${PREFIX}${DELIMITER}${type}`;