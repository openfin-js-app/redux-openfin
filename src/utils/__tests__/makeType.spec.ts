import makeType from '../makeType';

describe('makeType utils',()=>{
    it('works by default',()=>{
        expect(makeType('SAMPLE_ACTION_NAME')).toMatchSnapshot();
    })
});