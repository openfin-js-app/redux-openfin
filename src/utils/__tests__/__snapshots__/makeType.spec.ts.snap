// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`makeType utils works by default 1`] = `"REDUX_OPENFIN::SAMPLE_ACTION_NAME"`;
